moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -1,16 +1,5 @@
 import { FiltersCommonOperators } from "../../../components/GridTable/FiltersCommonOperators";
 
-/**
- * Array of operators that should be autocompleted
- * @constant
- * @type {array}
- * @default
- */
-export const AUTOCOMPLETE_OPERATORS = [
```

Comment:
Removing this config shifts setting what operators use the autocomplete input from being global to being specifically set in each field configuration. This way some fields like **Tags** and **Teams** can use the autocomplete with `string_contains_case_insensitive` too